_test_/components/first.test.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-unused-expressions */
+/* eslint-disable no-undef */
+import React from 'react';
+import { shallow } from 'enzyme';
+import App from '../../src/App';
+
+describe('first test', () => {
+  const wrapper = shallow(<App />);
+  it('should render the App', () => {
+    expect(wrapper.length).toEqual(1);
+    expect(wrapper.find('div')).toHaveLength;
+  });
+});
```

Comment:
It's high time we renamed this test file name and description. It's okay to use `app.test.js` and to update the describe to read `describe('app test, ...`

Reply:
Updated